core/src/main/java/org/bitcoinj/kits/WalletAppKit.java

```diff
@@ -196,6 +198,19 @@ public WalletAppKit restoreWalletFromSeed(DeterministicSeed seed) {
         this.restoreFromSeed = seed;
         return this;
     }
+    /**
```

Comment:
Is this a method comment? To me it's not immediately clear, also because there is a newline after the comment.

Reply:
It is supposed to be a method comment, yes.